src/components/ArtObjectPageComponents/PanelEnsemble/index.jsx

```diff
@@ -44,14 +44,16 @@ class PanelEnsemble extends Component {
     }
 
     const roomAndTitleText = getRoomAndTitleText(ensembleIndex);
+    const ensembleImageUrl =
+      this.props.object.ensembleImageUrl || ENSEMBLE_IMAGE_URL(ensembleIndex);
```

Comment:
Use the provided `ensembleImageUrl` on the object if it is available - otherwise fallback to the prior mechanism.